pkg/imgpkg/cmd/registry_flags.go

```diff
@@ -40,29 +39,6 @@ func (r *RegistryFlags) Set(cmd *cobra.Command) {
 
 	cmd.Flags().DurationVar(&r.ResponseHeaderTimeout, "registry-response-header-timeout", 30*time.Second, "Maximum time to allow a request to wait for a server's response headers from the registry (ms|s|m|h)")
 	cmd.Flags().IntVar(&r.RetryCount, "registry-retry-count", 5, "Set the number of times imgpkg retries to send requests to the registry in case of an error")
-
-	cmd.Flags().String("registry-azure-cr-config", "", "Path to the file containing Azure container registry configuration information. ($IMGPKG_REGISTRY_AZURE_CR_CONFIG)")
```

Comment:
In the docs we should make sure that we remove the existence of this flag as well

Reply:
Need to figure out the implications for the other auth providers as well

Reply:
The ACR library we're switching to (https://github.com/chrismellard/docker-credential-acr-env/) supports pulling authentication from the environment via `AZURE_CLIENT_ID`/`AZURE_CLIENT_SECRET`/`AZURE_TENANT_ID`. It's what's being used by [ko](https://github.com/google/ko/blob/691da5afb64e1d9204b250abc41fc738737b1ebb/pkg/commands/config.go#L47) and [cosign](https://github.com/sigstore/cosign/blob/516cc39f5582f46af9fc5b9081af08dba24441e7/cmd/cosign/cli/options/registry.go#L82-L90) so it seems like an acceptable choice